### PR TITLE
New version: WestWind.MarkdownMonster version 3.3.15.0

### PR DESCRIPTION
**Pull request by aisgbnok**
Checklist for Pull Requests
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/163858)

### CODE REVIEW

File: manifests/w/WestWind/MarkdownMonster/3.3.15.0/WestWind.MarkdownMonster.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- mmcli
+FileExtensions:
+- markdown
+- md
+- mdcrypt
+- mdproj
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
+    - PackageIdentifier: Microsoft.EdgeWebView2Runtime
```
Comment (stephengillie) [inline manifests/w/WestWind/MarkdownMonster/3.3.15.0/WestWind.MarkdownMonster.installer.yaml:23]:
```suggestion
    - PackageIdentifier: Microsoft.EdgeWebView2Runtime
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @aisgbnok,

Does this package have another dependency?
image

**Comment by aisgbnok** [comment]
> Hi @aisgbnok,
> 
> Does this package have another dependency? image

Hmm, not that I know of. Testing in Sandbox (W11 - 22635.3930):

image

They only list .Net and WebView2 on their website:
image
https://markdownmonster.west-wind.com/download

**Comment by aisgbnok** [comment]
@stephengillie

It looks like it is needed *only if you use the Save to PDF* feature.

File: manifests/w/WestWind/MarkdownMonster/3.3.15.0/WestWind.MarkdownMonster.installer.yaml (hunk 1 above)
Comment (aisgbnok) [reply manifests/w/WestWind/MarkdownMonster/3.3.15.0/WestWind.MarkdownMonster.installer.yaml:23]: I think this is a good suggestion to ensure all the features work as expected.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**